doc/backend/norms-api-v1.yaml

```diff
@@ -207,6 +208,48 @@ paths:
         "404":
           description: Norm not found.
 
+  /norms:
+    get:
+      tags:
+        - Norm work
+      summary: Get a list of all norm works
+      parameters:
+        - name: page
+          in: query
+          description: The page of the results
+          required: false
+          schema:
+            type: number
+        - name: size
+          in: query
+          description: The size of the pages
+          required: false
+          schema:
+            type: number
+      responses:
+        "200":
+          description: List of works.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  content:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/NormWork"
+                  page:
+                    type: object
+                    properties:
+                      size:
+                        type: number
```

Comment:
Also all this properties in the `page` object should be of type `integer`, not `number`